pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -91,6 +92,15 @@ type Plugin interface {
 	Name() string
 }
 
+// QueueSortPlugin is an interface that must be implemented by "QueueSort" plugins.
+// These plugins are used to sort pods in the scheduling queue. Only one queue sort
+// plugin may be enabled at a time.
+type QueueSortPlugin interface {
+	Plugin
+	// Less are used to sort pods in the scheduling queue.
+	Less(*internalqueue.PodInfo, *internalqueue.PodInfo) bool
```

Comment:
Is it better to name it `Sort`?

Reply:
I followed the KEP definition:

> These plugins are used to sort pods in the scheduling queue. A queue sort plugin essentially will provide a "less(pod1, pod2)" function. Only one queue sort plugin may be enabled at a time.

And the sort package in golang also prefers `Less` in this scenario

Reply:
All right, understand.